### ISSUE TITLE
forgetting to export a class doesn't create an error

### ISSUE THREAD

**Issue by xenoterracide** [aurelia/aurelia#846]
`npx makes aurelia` (=choose typescript (maybe I haven't tested in a non typescript project)

add a component and import it 

`hello.html`
```html
<p>hello world</p>
```
```ts
class Hello {
}
```
`my-app.html`
```html
<import from='./hello'></import>
...
<a goto='hello'>hello</a>
...
  <au-viewport class="container" default="index" fallback="missing"></au-viewport>
```

navigating to hello will result in reaching the fallback route. This is because `Hello` is not exported, no error is thrown.

**Comment by Sayan751** [comment]
@3cp Can you please take look at this?

**Comment by 3cp** [comment]
If developers want something to check if they forgot export, probably eslint is a better choice, because syntax wise there is nothing wrong about js file without export.

**Comment by Sayan751** [comment]
@3cp You are right. Just thinking it aloud, would it be possible to raise an error when the `hello` from `<import from='./hello'>` does not export anything? Or, specifically in this case, any Aurelia resources? If that's doable while statically analysing the templates, I think we should do it.

**Comment by 3cp** [comment]
It's doable, but requires much more effort. Currently conventions plugin is implemented very cheaply, it was designed as a temporary solution before AOT lands.

**Comment by Sayan751** [comment]
Understood. No stress. This is tagged to v2.x milestone anyway. We might have more issues to address for beta/rc.

**Comment by 3cp** [comment]
Wait a min, I think it's talking about the runtime should throw an error instead of silently redirect to fallback route.
